document-portal/permission-db.c

```diff
@@ -734,7 +734,7 @@ permission_db_update (PermissionDb *self)
 
   g_return_if_fail (PERMISSION_IS_DB (self));
 
-  root = gvdb_hash_table_new (NULL, NULL);
+  g_autoptr(GHashTable) root = root = gvdb_hash_table_new (NULL, NULL);
```

Comment:
The double self-assignment doesn't seem right here. Also, minor nit, mind moving the `g_autoptr(GHashTable) root = NULL` declaration to the beginning of the function?